crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -753,6 +821,20 @@ impl AsBindGroupShaderType<StandardMaterialUniform> for StandardMaterial {
                 flags |= StandardMaterialFlags::DIFFUSE_TRANSMISSION_TEXTURE;
             }
         }
+
+        #[cfg(feature = "pbr_multi_layer_material_textures")]
+        {
+            if self.clearcoat_texture.is_some() {
+                flags |= StandardMaterialFlags::CLEARCOAT_TEXTURE;
+            }
```

Comment:
nitpick - these flags should probably be added in the prepass as well in case anybody wants to use them

Reply:
I don't know what this means. The prepass doesn't care about any of these flags, and we don't do this anywhere else (grepping for e.g. `BASE_COLOR` brings up nothing related to prepasses, except for alpha discard, which isn't relevant here).

Reply:
Yeah I wouldn't add them. Sometime soon we should revisit how pipeline keys are managed however.